.github/workflows/test.yml

```diff
@@ -37,7 +37,7 @@ jobs:
         ruby-version: ${{ matrix.ruby }}
         bundler-cache: true
     - name: Run test
-      run: bundle exec rake test
+      run: bundle exec rake
```

Comment:
I would make `test_in_isolation` a separate step. It'd be easier to understand why it failed.